tools/js-tools/package.json

```
@@ -17,6 +17,9 @@
 		"@babel/eslint-parser": "7.17.0",
 		"@octokit/auth-token": "2.4.5",
 		"@octokit/rest": "18.6.7",
+		"@rushstack/eslint-patch": "^1.1.1",
+		"@typescript-eslint/eslint-plugin": "^5.16.0",
+		"@typescript-eslint/parser": "^5.16.0",
```

I suspect renovate will come along and want to do
```suggestion
		"@rushstack/eslint-patch": "1.1.1",
		"@typescript-eslint/eslint-plugin": "5.16.0",
		"@typescript-eslint/parser": "5.16.0",
```
here.

---

Done. Thank you.